simulator/docs/env-variables.md

```diff
@@ -0,0 +1,36 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator. 
+
+Please refer [docker-compose.yml](./docker-compose.yml) as an example
+use.
+
+`Port`: (required) This is the port number on which kube-scheduler-simulator
+server is started.
+
+`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for
+etcd. The simulator runs kube-apiserver internally, and the
+kube-apiserver uses this etcd.
+
+`FrontendURL`: This URL represents the URL web UI started. The
+simulator and internal kube-apiserver set the origin as the allowed
+origin for `CORS_ALLOWED_ORIGIN_LIST`.
+
+`KUBECONFIG`: This is for the beta feature "Existing cluster
+Importing". This variable is used to find Kubeconfig required to
+access your cluster for importing resources to scheduler simulator.
+
+`KUBE_API_HOST`: This is the host of kube-apiserver which the
+simulator starts internally.
```

Comment:
```suggestion
`KUBE_API_HOST`: This is the host of kube-apiserver which the
simulator starts internally. Its default value is `3131`.
```

Reply:
And `KUBE_API_PORT` env should be mentioned as well. `KUBE_API_PORT` is the port of kube-apiserver.

Reply:
Updated.